CHANGELOG.adoc

```diff
@@ -39,6 +39,7 @@ Currently it is not possible to compute different values for width and height.
 - https://github.com/eclipse-sirius/sirius-components/issues/134[#134] [workbench] The internal API of the workbench is now ready to accept features leveraging a multi-selection
 - [form] Add a tooltip to always make the full label available
 - [core] Customize the Spring `ObjectMapper` instead of creating a brand new one from scratch in our `ObjectMapperConfiguration`
+- https://github.com/eclipse-sirius/sirius-components/issues/896[#896] [view] Allow to make specific changes before and after the layout
```

Comment:
This is not related to `[view]` which is our DSL to define diagram descriptions in the web browser (sirius-web-view and sirius-web-view-edit), it is related to `[diagram]`.

Reply:
Done